client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less

```diff
@@ -0,0 +1,74 @@
+.bio-vo-tab-content {
+  .bio-vo-label {
+    font-size: 14px !important;
+  }
+
+  .bio-vo-search .cds--search-input,
+  .bio-vo-search .cds--search-close {
+    block-size: 2rem !important;
+  }
+
+  .bio-vo-search .cds--search-close:before {
+    block-size: calc(100% - 2.5px) !important;
+    inset-block-start: 1.5px !important;
+  }
+
+  .bio-vo-empty-search {
+    h3 {
+      font-size: 14px !important;
+      font-weight: bold !important;
+      margin-bottom: 7px !important;
+    }
+
+    p {
+      font-size: 14px !important;
+    }
+  }
+
+  .bio-vo-search-container,
+  .bio-vo-search-container>*+* {
+    border-color: var(--color-grey-225-10-75) !important;
+  }
+
+  .bio-vo-tab-coloumn {
+    display: flex !important;
+
+    .bio-vo-element-list,
+    .bio-vo-variable-table {
+      flex-grow: 1;
+    }
+  }
+
+  .bio-vo-variable-table {
+    border-left: none !important;
+    border-right: none !important;
+  }
+
+  .bio-vo-tab-coloumn:has(.bio-vo-variable-table),
+  .bio-vo-tab-coloumn:has(.bio-vo-empty-search) {
+    border-left: 1px solid var(--color-grey-225-10-75) !important;
+  }
+
+  .bio-vo-empty-search {
+    align-self: center !important;
+    margin: auto !important;
+    padding: 0 !important;
+  }
+
+  th,
```

Comment:
```suggestion
  .bio-vo-variable-table th,
```

Reply:
I realized we already apply the whole fixes locally to the variables panel `.bio-vo-tab-content`, so no additional fixes should be needed.